js/ui/date_box/ui.date_view.js

```diff
@@ -232,13 +232,10 @@ var DateView = Editor.inherit({
         if(monthRoller) {
             this._createRollerConfig(ROLLER_TYPE.month);
             var monthRollerConfig = this._rollerConfigs[ROLLER_TYPE.month];
-
-            if(monthRollerConfig.displayItems.length !== monthRoller.option("items").length) {
-                monthRoller.option({
-                    items: monthRollerConfig.displayItems,
-                    selectedIndex: monthRollerConfig.selectedIndex
-                });
-            }
+            monthRoller.option({
```

Comment:
This PR leads to the changing of the monthRoller options on change every roller. E.g. you change a day from 10 to 11 and monthRoller will be changed too